### DISCUSSION TITLE
dc direct delete is awesome

### DISCUSSION THREAD

**Discussion by goyalyashpal** [doublecmd/doublecmd#795] [General]
DC 's direct delete just deleted in a jify what others couldn't since years:
I tried doing from windows explorer, mucommander, from command line, with elevated privileges, even with disk cleaners/thrashers... nothing worked....

So, what does DC's direct delete do that none other did?
How can it rename and delete those files?

`.freedownloadmanager` aka FDM was the download manager i used in the past... 
i guess it couldn't complete the files, but then too, there are other incomplete download files which get deleted quickly by normal methods too.

| software |
| -- |
| **Explorer** <br/><br/><img src="https://user-images.githubusercontent.com/19423063/216813613-d7517a6a-f153-48f6-9c65-bee3d63e4132.png" width=300px/> <img src="https://user-images.githubusercontent.com/19423063/216814201-562d7f3d-41fb-48ec-934c-8b24e0f81cf9.png" width=500px/> |
| **mucommander** <sup>ignore the name shown here, i guess it's picking up the name dc renamed, and it's showing same name for all files</sup><br/><br/><img src="https://user-images.githubusercontent.com/19423063/216815645-443ba335-c35e-475f-9d0a-979190e12854.png" width=400px/><img src="https://user-images.githubusercontent.com/19423063/216813861-6a134c7e-94ba-4601-9d27-89c1a95d92ea.png" width="400px"/> |
| **doublecmd** <br/><br/><img src="https://user-images.githubusercontent.com/19423063/216812873-9c02baa2-32ed-42ca-a16a-ba4e87a26987.png" width=409px/><img src="https://user-images.githubusercontent.com/19423063/216815721-400517f7-07e5-483e-af5c-90feebc07ee3.png" width=409px/> |

**Comment by alexx2000** [comment]
Explorer does not like filenames with a dot or space at the end.

**Reply by goyalyashpal** [reply]
> Explorer does not like filenames with a dot or space at the end.

woww, nice.

glad i didn't delete all of those files. 

Yes, i removed the trailing space at end by renaming in dc, then deleted the renamed file via explorer, and it deleted it too.

thanks for sharing.
